crates/egui_demo_app/src/backend_panel.rs

```diff
@@ -218,13 +218,10 @@ impl BackendPanel {
                 )
                 .on_hover_text("Physical pixels per point.");
 
-            if response.drag_released() {
+            if !response.dragged() {
                 // We wait until mouse release to activate:
                 ui.ctx().set_pixels_per_point(*pixels_per_point);
                 reset = true;
-            } else if !response.is_pointer_button_down_on() {
-                // When not dragging, show the current pixels_per_point so others can change it.
-                reset = true;
```

Comment:
The new code will call `set_pixels_per_point` each frame, which means that nothing else can change the `pixels_per_point`. For instance: dragging the eframe window to another screen should change the pixels_per_point, but the new code will forbid that. That was the point of the comment:

`// When not dragging, show the current pixels_per_point so others can change it.`